CHANGES.md

```diff
@@ -1,5 +1,152 @@
+of the Linkerd CRDs from the main installation process. With the CLI, you'll
+need to `linkerd install --crds` before running `linkerd install`; with Helm,
+you'll install the new `linkerd-crds` chart, then the `linkerd-control-plane`
+chart. These charts are now versioned using [SemVer](https://semver.org)
+independently of Linkerd releases. For more information, see the
+[upgrade notes][upgrade-2120].
+
+**Upgrade notes**: Please see the [upgrade instructions][upgrade-2120].
+
+* Proxy
+  * Added a `config.linkerd.io/shutdown-grace-period` annotation to limit
+    the duration that the proxy may wait for graceful shutdown
+  * Added a new `iptables-nft` mode for the `proxy-init` initContainer
+  * Added support for non-HTTP traffic forwarding within the mesh in `ingress` mode
+  * Added the `/logs.json` log stream endpoint
+  * Added a new `process_uptime_seconds_total` metric to track proxy uptime in seconds
+  * Added support for dynamically discovering policies for ports that are not documented
+    in a pod's `containerPorts`
+  * Added support for route-based inbound HTTP metrics (`route_group`/`route_kind`/`route_name`)
+
+* Control Plane
+  * Added support for per-route policy by supporting AuthorizationPolicy
+    resources which target HttpRoute resources
+  * Added support for bound service account token volumes for the control plane
+    and injected workloads
+  * Removed kube-system exclusions from watchers to fix service discovery
+    for workloads in the kube-system namespace (thanks @JacobHenner!)
+  * Updated healthcheck to ignore `Terminated` state for pods (thanks
+    @AgrimPrasad!)
+  * Updated the default policy controller log level to `info`; the controller
+    will now emit INFO level logs for some of its dependencies
+  * Added probe authorization by default, allowing clusters that use a
+    default `deny` policy to not explicitly need to authorize probes
+  * Fixed an issue where the proxy-injector would break when using
+    `nodeAffinity` values for the control plane
+  * Fixed an issue where certain control plane components were not restarting
+    as necessary after a trust root rotation
+
+* CLI
+  * Fixed the `linkerd check` command crashing when unexpected pods are found
+    in a Linkerd namespace
+  * Updated the `linkerd authz` command to support AuthorizationPolicy and
+    HttpRoute resources
+  * Updated `linkerd check` to allow RSA signed trust anchors (thanks
+    @danibaeyens!)
+  * `linkerd install --crds` must be run before `linkerd install`
+  * `linkerd upgrade --crds` must be run before `linkerd upgrade`
+  * Fixed invalid yaml syntax in the viz extension's tap-injector template
+    (thanks @wc-s!)
+  * Fixed an issue where the `--default-inbound-policy` setting was not
+    being respected
+  * Added support for AuthorizationPolicy and HttpRoute to `viz authz` command
+  * Added support for AuthorizationPolicy and HttpRoute to `viz stat` command
```

Comment:
other recent changelogs for stable releases (e.g. [2.11.0](https://github.com/linkerd/linkerd2/blob/939470e57b0c2108e0c3b7fabecc5658f84d0752/CHANGES.md#stable-2110) put `viz` changes in their own bullet point under the "Extensions" bullet point, instead of mixing them in with the "CLI", "Helm", etc bullets the way we do here. do we want to do that for these release notes as well?

Reply:
@hawkw @kleimkuhler Putting the CLI subcommand updates into the CLI section makes sense to me.